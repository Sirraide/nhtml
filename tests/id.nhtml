div#foo {
    #bar (asfasdfadfadsf)
    #baz.foo (asfasdfadfadsf)
    .foo#qux (asfasdfadfadsf)
}
