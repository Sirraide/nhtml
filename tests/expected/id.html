<div id="foo">
    <div id="bar">asfasdfadfadsf</div>
    <div class="foo" id="baz">asfasdfadfadsf</div>
    <div class="foo" id="qux">asfasdfadfadsf</div>
</div>
